apps/machine/zynqmp_r5/platform_info.c

```diff
@@ -229,6 +303,7 @@ platform_create_rpmsg_vdev(void *platform, unsigned int vdev_index,
 		goto err2;
 	}
 	xil_printf("initializing rpmsg vdev\r\n");
+	report_setup(role);
```

Comment:
Additional seems that there are 2 times in code` xil_printf("initializing rpmsg vdev\r\n");`
could you take opportunity to clean it

Reply:
sure.